File: app/src/main/java/in/completecourse/questionbank/LoginActivity.java
package in.completecourse.questionbank;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import in.completecourse.questionbank.fragment.EasyLoginFragment;
import in.completecourse.questionbank.helper.HelperMethods;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup);
        HelperMethods.INSTANCE.loadFragment(new EasyLoginFragment(), this);
    }

}

File: app/src/main/java/in/completecourse/questionbank/LoginActivity.kt
package `in`.completecourse.questionbank

import `in`.completecourse.questionbank.fragment.EasyLoginFragment
import `in`.completecourse.questionbank.helper.HelperMethods
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup)
        HelperMethods.loadFragment(EasyLoginFragment(), this)
    }
}